go.mod

```diff
@@ -42,17 +42,19 @@ require (
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect; indirects
 	github.com/spf13/pflag v1.0.5
 	github.com/spf13/viper v1.7.1
-	github.com/stretchr/testify v1.6.1
+	github.com/stretchr/testify v1.7.0
```

Comment:
note: this line has been in https://github.com/cosmos/cosmos-sdk/commit/334b2238d84191df89acea9073a621dda57ea9c0 in master, but not backported. So it's updated in this PR.

Reply:
It was not backported because it's not related to any bug / security issue.